### ISSUE TITLE
Implement day/night mode on Edge devices

### ISSUE THREAD

**Issue by TheNinth7** [openhab/openhab-garmin#256]
The focus indicator on the Edge 850, and presumably also on the 550, assumes a white background. Therefore, the menu should be implemented using day mode colors.
Support for night mode may be possible, but this needs to be verified, in particular whether the focus indicator itself supports night mode.

**Comment by TheNinth7** [comment]
All of the above is completed.

One new point:

- Play/Pause icons in the menu item should be in state color.
